.github/workflows/playwright.yml

```diff
@@ -38,9 +38,6 @@ jobs:
           npm ci
           npx playwright --version
 
-      - name: Install Playwright Browsers
-        run: npx playwright install --with-deps
-
```

Comment:
The Docker image used has includes the Playwright browsers and browser system dependencies ([link to source](https://playwright.dev/docs/docker#introduction))